### ISSUE TITLE
some deprecated methods in oauth2

### ISSUE THREAD

**Issue by shinriyo** [CORDEA/oauth#12]
I run `nim c -d:ssl -r gitlab.nim` in example folder.

`oauth2.nim` has some deprecated methods.

```
/Users/shinriyo/.nimble/pkgs/oauth-0.5/oauth2.nim(116, 40) Warning: use HttpPost instead; httpPost is deprecated [Deprecated]
/Users/shinriyo/.nimble/pkgs/oauth-0.5/oauth2.nim(116, 14) Warning: request is deprecated [Deprecated]
/Users/shinriyo/.nimble/pkgs/oauth-0.5/oauth2.nim(239, 60) Warning: use HttpGet instead; httpGet is deprecated [Deprecated]
/Users/shinriyo/.nimble/pkgs/oauth-0.5/oauth2.nim(242, 14) Warning: request is deprecated [Deprecated]
```

**Comment by shinriyo** [comment]
sorry. I guess some deprecated was already fix in `develop` branch...
https://github.com/CORDEA/oauth/commit/b66d59888b5e72dee5dab645b7d415866362bff5#diff-b476939b66bb31ca38fee6f83a7d5ad5

I hope merge to `master`.

**Comment by CORDEA** [comment]
Thanks for reporting :)
OK, I will merge to master and release it.

**Comment by shinriyo** [comment]
@CORDEA Thank you. I am looking forward to that.

**Comment by CORDEA** [comment]
Now, I released [v0.6](https://github.com/CORDEA/oauth/releases/tag/v0.6). v0.6 requires nim v0.18.* because I fixed deprecated warning of random together. 
Please let me know if you find any problems ;)
